package.json

```diff
@@ -18,7 +18,7 @@
     "dist:colors:prettier-sketchpalette": "prettier ./dist/*.sketchpalette --write --parser json",
     "dist:colors:ase": "node ./scripts/ase-encode.js",
     "dist:colors:clr:excludeFromCI": "ase2clr ./dist/colors.ase ./dist/colors.clr",
-    "dist:colors:rename-palettes": "renamer --find colors --replace Polaris ./dist/colors.{ase,clr,sketchpalette}",
+    "dist:colors:rename-palettes:excludeFromCI": "renamer --find colors --replace Polaris ./dist/colors.{ase,clr,sketchpalette}",
```

Comment:
`renamer` v1.0.0 is less forgiving than its previous version, meaning this command was failing. It shouldn't be a big deal since `yarn dist-ci` is mostly about testing the other steps (Gulp tasks and such) compile successfully.